fxa-auth-db-server/test/backend/remote.js

```diff
@@ -249,7 +249,7 @@ module.exports = function(cfg, server) {
           respOk(t, r)
           var sessions = r.obj
           t.equal(sessions.length, 1, 'sessions contains one item')
-          t.equal(Object.keys(sessions[0]).length, 9, 'session has nine properties')
+          t.equal(Object.keys(sessions[0]).length, 16, 'session has correct properties')
```

Comment:
Same as above, maybe an additional test case to verify correct properties are set.